static/scripts/newblocks.js

```diff
@@ -666,7 +666,7 @@ Blockly.JavaScript["experimental_raw_value"] = function (block) {
 };
 
 Blockly.JavaScript["comment"] = function (block) {
-  return "//" + block.getFieldValue("text") + "\n";
+  return "//" + block.getFieldValue("text").replaceAll('\n',' ') + "\n";
```

Comment:
Why do we need to use this?

Reply:
If you input a new line in the comment it will literally add a new line to the code. If this happens the rest of the comment isn't escaped. A possible alternative solution would be to escape the entire comment. Not sure which translation to JavaScript Code is more fitting.